crates/api/src/v1/package.rs

```diff
@@ -58,9 +53,10 @@ pub struct PublishRecordRequest<'a> {
 
 /// Represents a package record API entity in a registry.
 #[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct PackageRecord {
     /// The identifier of the package record.
-    pub id: RecordId,
+    pub record_id: RecordId,
```

Comment:
I don't think we discussed this change; I'm not opposed to it but I'd like to be consistent across APIs, e.g. changing package IDs to `package_id` as well.

Reply:
Yeah, more explicit is better. I'll update `package_id` as well.

Reply:
Updated `package_id` as well to be more consistent throughout.